zipkin-server/src/main/java/zipkin/server/ActuateCollectorMetrics.java

```diff
@@ -41,75 +50,85 @@
  *
  * See https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-metrics.html
  */
-final class ActuateCollectorMetrics implements CollectorMetrics {
-
-  private final CounterService counterService;
-  private final GaugeService gaugeService;
+@UsesJava8
```

Comment:
mentioned offline, this is perfectly fine as our server is java 8 since the beginning